apps/web/components/SubpageMainContent/SubpageMainContent.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode } from 'react'
+import { useWindowSize, useIsomorphicLayoutEffect } from 'react-use'
+import { theme } from '@island.is/island-ui/theme'
+import {
+  Box,
+  GridContainer,
+  GridRow,
+  GridColumn,
+} from '@island.is/island-ui/core'
+
+interface SubpageMainProps {
+  main: ReactNode
+  image?: ReactNode
+}
+
+export const SubpageMainContent: FC<SubpageMainProps> = ({ main, image }) => {
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = React.useState(false)
+
+  useIsomorphicLayoutEffect(() => {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
+
+  return (
+    <Box>
+      {isMobile ? (
+        <GridContainer>{main}</GridContainer>
+      ) : (
+        <Box>
+          {image ? (
+            <GridContainer>
+              <GridRow>
+                <GridColumn span="9/12">{main}</GridColumn>
+                <GridColumn span="3/12">{image}</GridColumn>
+              </GridRow>
+            </GridContainer>
+          ) : (
+            <GridContainer>{main}</GridContainer>
+          )}
+        </Box>
+      )}
+    </Box>
```

Comment:
this could probably be simplified to something like this
```suggestion
    <GridContainer>
      <GridRow>
        <GridColumn span={image ? "9/12" : '12/12'}>{main}</GridColumn>
        {image  && <GridColumn span="3/12" hiddenBelow='md'>{image}</GridColumn>}
      </GridRow>
    </GridContainer>
```
and you could remove the js logic for breakpoints

Reply:
Yes, this works better. I did make the main span responsive if there is an image so we wouldn't get some unused whitespace to the right when the image gets hidden on mobile.